docs/architecture.md

```
@@ -0,0 +1,68 @@
+# System level architecture
+
+This document describes all components of BacPack system and how they work together.
+
+## Components
+
+In a nutshell, BacPack system contains these components:
+
+ - Packager
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)